Turf.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
   # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
   s.name = "Turf"
-  s.version = "1.2.0"
+  s.version = "2.0.0-alpha.2"
```

Comment:
I’ll take care of increasing the version to v2.0.0-alpha.3 in a separate PR for clarity.